### DISCUSSION TITLE
Automated AI Meme Generation for Work-Related Situations

### DISCUSSION THREAD

**Discussion by julialawrence** [ministryofjustice/analytical-platform#2639] [Firebreak Winter 23]
Details in https://mojdt.slack.com/archives/C0689TSK7NJ/p1701688556214669

**Comment by bagg3rs** [comment]
We need some metric baselines to workout how much efficiencies in memetime we would be saving the team

**Reply by julialawrence** [reply]
That sounds like another great firebreak idea!

**Comment by jacobwoffenden** [comment]
<img src="https://media2.giphy.com/media/a93jwI0wkWTQs/giphy.gif"/>
